qiskit_experiments/calibration_management/backend_calibrations.py

```diff
@@ -43,8 +44,25 @@ class BackendCalibrations(Calibrations):
     __qubit_freq_parameter__ = "qubit_lo_freq"
     __readout_freq_parameter__ = "meas_lo_freq"
 
-    def __init__(self, backend: Backend):
-        """Setup an instance to manage the calibrations of a backend."""
+    def __init__(
+        self,
+        backend: Backend,
+        basis_gates: Optional[List[str]] = None,
+        library: Type[BasisGateLibrary] = None,
+        library_options: Optional[Dict] = None,
```

Comment:
can this be kwargs? or do we need to write dictionary?

Reply:
We can go the kwargs route. See here: d10d155.